packages/pharos/src/components/image-card/pharos-image-card.ts

```diff
@@ -127,6 +127,13 @@ export class PharosImageCard extends LitElement {
 
   private _handleImageHover(event: Event): void {
     this._title['_hover'] = event.type === 'mouseenter';
+
+    const mouseEvent = new CustomEvent(
+      event.type === 'mouseenter'
+        ? 'pharos-image-card-image-mouseenter'
+        : 'pharos-image-card-image-mouseleave'
+    );
+    this.dispatchEvent(mouseEvent);
   }
```

Comment:
Not a blocker but I wonder if it makes sense to have 2 different handlers (one for mouse enter and other for mouse leave). Seems like that would simplify a couple conditionals without too much code duplication 🤷

Reply:
yeah i could see that going both ways. 
the ` _handleImageHover` event already existed for the title hover so i just followed that path 🤔

Reply:
updated. i would say it looks cleaner